classes/file/TemporaryFileManager.inc.php

```diff
@@ -57,6 +57,16 @@ function deleteFile($fileId, $userId) {
 	}
 
 	/**
+	 * Delete a file by path.
+	 * For temporary files that are not in the DB, e.g. exported xml files.
+	 * @param $filePath string the location of the file to be deleted
+	 * @return boolean returns true if successful
+	 */
+	function deleteFileByPath($filePath) {
```

Comment:
I agree that persisting to the database isn't useful in this case, but it does permit the `TemporaryFileManager` to take care of managing the file storage, cleanup, etc. The less we re-implement those concerns the better, I think.

For one-shot cases like this there might be a role for a simpler file manager class, but the extra overhead from persisting to the database is so little that I'm not sure it's worth the extra code.